src/transformers/dependency_versions_check.py

```diff
@@ -0,0 +1,11 @@
+from .dependency_versions_table import deps
+from .utils.versions import require_version_core
+
+
+# define which module versions we always want to check at run time (usually the ones defined in `install_requires` in setup.py)
+pkgs_to_check_at_runtime = "python tokenizers tqdm regex numpy".split()
```

Comment:
`tokenizers` does not have to be installed no? It's not mandatory IMO.

Reply:
(nit) think it's more readable to directly write `for k in "python tokenizers tqdm regex numpy".split()` ....

Reply:
It's still in the `setup.py` but we've indeed done a lot of efforts for it not to be required as some machines cannot have `tokenizers` installed yet. We're leaving it in the `setup.py`, and expect users that have an issue with `tokenizers` to install `transformers` without dependencies.

I would be in favor of not checking that `tokenizers` is available at runtime.

Reply:
Hadn't seen your comment Sylvain, I agree with you.

Reply:
Done.